client/src/test/java/org/apache/rocketmq/client/impl/factory/MQClientInstanceTest.java

```diff
@@ -42,6 +47,14 @@
     private MQClientInstance mqClientInstance = MQClientManager.getInstance().getOrCreateMQClientInstance(new ClientConfig());
     private String topic = "FooBar";
     private String group = "FooBarGroup";
+    private ConcurrentMap<String, HashMap<Long, String>> brokerAddrTable = new ConcurrentHashMap<String, HashMap<Long, String>>();
+
+    @Before
+    public void init() throws Exception {
+        Field field = MQClientInstance.class.getDeclaredField("brokerAddrTable");
```

Comment:
mockito and jmockit has native support for the reflection

Reply:
done

Reply:
My idea was to remind you to use jmockit's built-in reflection method to mock private implementation. For some tools, do not use the internal package. These codes are created mainly by its usage, which is not suitable for direct import in projects. This is good practice. For this problem, The value of the two implementations（mock or inject reflection） is not much different. I hope you could pay attention to this point in the future :-)

Reply:
Thanks for your reminding！I will pay attention to it.